process/suspension.html

```diff
@@ -58,10 +58,17 @@ <h4>Nearby</h4>
 </ul>
 <p>If the chair, team contact, CEO, or COO is the one accused of unacceptable behavior, they should not be involved in the decision to ban.</p>
 
-<p>2. Following violation of the <a href="/policies/code-of-conduct/">W3C Code of Conduct</a>, at least one warning, and at least one subsequent violation of the Code of Conduct, chairs, or team, may temporarily or permanently suspend participation by a person who has not formally joined the group.
+<p>2. Following violation of the <a href="/policies/code-of-conduct/">W3C Code of Conduct</a>, Chairs may gently remind about minor Code of Conduct violations, such as inadvertent microaggressions in public (meeting, email list) using language such as, "As a reminder, Person's pronoun's are they/them." This shows support for the microagressed person and demonstrates what will and won't be tolerated without pointing a finger at an individual. All other discussions of code of conduct violation should be in private unless there is an emergency (such as active assault in person). If a chair must remind a participant about the Code of Conduct:
 </p>
-
-<p>3. Following a warning, a W3C group chair and team contact may propose to suspend or remove a participant from a single group - typically for egregious and repetitive Code of Conduct violations - by making a request to the CEO.</p>
+<ul>
+<li>Mention that we have a mutual agreement to operate under the Code of Conduct</li>
+<li>Point to the specific part of the Code that has been violated</li>
+<li>Point to the section of the Code of Conduct that discusses what to do if <a href="https://www.w3.org/policies/code-of-conduct/#mistake">you've done something improper</a></li>
+<li>Ask what support is needed to engage appropriately</li>  
+</ul>
+<p>Note that the section on <a href="https://www.w3.org/policies/code-of-conduct/#safety-versus-comfort">Safety versus Comfort</a> includes information about prioritizing a Code of Conduct complaint over a response to a complaint.</p>
+<p>After at least one warning and at least one subsequent violation of the Code of Conduct, chairs, or team, may temporarily or permanently suspend participation by a person who has not formally joined the group.</p>
+      <p>3. Following a warning, a W3C group chair and team contact may propose to suspend or remove a participant from a single group - typically for egregious and repetitive Code of Conduct violations - by making a request to the CEO.</p>
```

Comment:
Given that "the Code of Conduct" is the focus of this section, I think it can almost always be referenced as simply "the Code" — or it should always be referenced as "the Code of Conduct".

GitHub won't let me make a second suggestion, targeting a single line, so I'm putting this into the large suggestion. These space-surrounded hyphens should be space-surrounded em-dashes, and there's no reason to indent this `<p>` start.

Tangentially, I'm wondering why numbered paragraphs (and larger) aren't being handled as ordered lists. The drop-indent makes such lists much easier to read and interpret logically, especially when their content is more than a line or two of text.
```suggestion
<li>Mention that we have a mutual agreement to operate under the Code</li>
<li>Point to the specific part of the Code that has been violated</li>
<li>Point to the section of the Code that discusses what to do if <a href="https://www.w3.org/policies/code-of-conduct/#mistake">you've done something improper</a></li>
<li>Ask what support is needed to engage appropriately</li>  
</ul>
<p>Note that the section on <a href="https://www.w3.org/policies/code-of-conduct/#safety-versus-comfort">Safety versus Comfort</a> includes information about prioritizing a Code complaint over a response to a complaint.</p>
<p>After at least one warning and at least one subsequent violation of the Code of Conduct, chairs, or team, may temporarily or permanently suspend participation by a person who has not formally joined the group.</p>
<p>3. Following a warning, a W3C group chair and team contact may propose to suspend or remove a participant from a single group — typically for egregious and repetitive Code violations — by making a request to the CEO.</p>
```